.drone.yml

```diff
@@ -360,7 +360,7 @@ services:
     image: microsoft/mssql-server-linux:latest
     environment:
       - ACCEPT_EULA=Y
-      - SA_PASSWORD=M$wantsaSecurePassword
+      - SA_PASSWORD=M$wantsaSecurePassword1
```

Comment:
I think you need to escape `$` character otherwise it tries to connect with password `MwantsaSecurePassword1`

Reply:
but maybe not it this file but in app.ini

Reply:
Thanks for pointing that out, I thinks thats the issue.